src/query/functions/utils/params.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package utils
+
+// StaticParams is a simple string Params implementation, useful for when no data other than OpType is needed.
+type StaticParams string
+
+// String simply returns s.
+func (s StaticParams) String() string {
+	return string(s)
```

Comment:
Do you need to make this a `string()` or can you just return `s`? Might be a noob go question..

Reply:
Nah totally reasonable question. https://play.golang.org/p/HOEP-EYYyNC doesn't compile unless you explicitly mark the conversion (though my understanding is that it's a completely compile time check with no runtime implications, since the underlying types are identical) https://golang.org/ref/spec#Conversions